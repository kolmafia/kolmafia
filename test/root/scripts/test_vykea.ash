string none = $vykea[none].to_string();
print(none);
vykea dishrack = $vykea[level 1 dishrack];
print(dishrack.image);
