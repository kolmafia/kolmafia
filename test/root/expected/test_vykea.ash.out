none
vykfurn6.gif
